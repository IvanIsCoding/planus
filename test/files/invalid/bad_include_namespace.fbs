include "helpers/bad_include_namespace_helper.fbs";

namespace Foo;

table Foo {}