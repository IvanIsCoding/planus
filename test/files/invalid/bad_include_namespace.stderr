[0m[1m[38;5;9merror[0m[1m: Overlapping declarations[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/bad_include_namespace.fbs:5:1
  [0m[34mâ”‚[0m
[0m[34m5[0m [0m[34mâ”‚[0m table Foo {}
  [0m[34mâ”‚[0m [0m[34m---------[0m [0m[34mfirst definition was here[0m
  [0m[34mâ”‚[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/helpers/bad_include_namespace_helper.fbs:3:1
  [0m[34mâ”‚[0m
[0m[34m3[0m [0m[34mâ”‚[0m table Foo {}
  [0m[34mâ”‚[0m [0m[34m---------[0m [0m[34msecond definition was here[0m

Error: Bailing because of errors
